test/functional/p2p_invalid_tx.py

```diff
@@ -77,10 +82,29 @@ def run_test(self):
                 reject_reason=template.reject_reason,
             )
 
+            # Only for bad size test, uses testmempoolaccept to make sure something
+            # just big enough passes the policy check without modifying mempool for rest of test
+            if BadTxTemplate.__name__ == "SizeTooSmall":
+                found_badsize = True
+                # Double-check via same interface to cross t's and dot i's
+                assert_equal(node.testmempoolaccept(rawtxs=[tx.serialize().hex()])[0]['reject-reason'], 'tx-size-small')
+                assert_equal(len(tx.serialize()), MIN_STANDARD_TX_NONWITNESS_SIZE - 1)
+
+                # Make it one non-witness byte larger
+                tx.vout[0].nValue = block1.vtx[0].vout[0].nValue - 1000 # 1k sat fee
+                tx.vout[0].scriptPubKey = DUMMY_MIN_OP_RETURN_SCRIPT
+                assert_equal(len(tx.serialize()), MIN_STANDARD_TX_NONWITNESS_SIZE)
+                result = node.testmempoolaccept(rawtxs=[tx.serialize().hex()])
+                assert result[0]['allowed']
```

Comment:
Maybe move this to `mempool_accept`? The purpose of both files looks very similar but I guess mempool_accept feels a bit closer to host a manual check?